tbb.spec

```
@@ -5,11 +5,13 @@
 %define github_user oneapi-src
 %define github_repo oneTBB
 Source: git+https://github.com/%{github_user}/%{github_repo}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{branch}-%{tag}.tgz
+Patch0: https://patch-diff.githubusercontent.com/raw/oneapi-src/oneTBB/pull/782.patch
```

```suggestion
Patch0: https://raw.githubusercontent.com/cms-sw/cmsdist/3140f3134a3b50a02ad9192f228b7f4b06a38747/tbb-782.patch
```

The original patch does not apply cleanly.

---

Or just
```suggestion
Patch0: tbb-782.patch
```